### PR TITLE
More XEPs in the DOAP file

### PR DESCRIPTION
**Pull request by Zash**
Might still not be complete. Might be wrong. Missing details on most of the ones added.

### CODE REVIEW

File: dino.doap.in [hunk 1]
```diff
@@ -202,5 +273,26 @@
...
+    </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0398.html" />
+        <xmpp:status>complete</xmpp:status>
+      </xmpp:SupportedXep>
+    </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0402.html" />
```
Comment (fiaxh) [inline dino.doap.in:289]: Dino has code that does 402, but doesn't actually use it in practive yet. Same for 0223.

File: dino.doap.in [hunk 2]
```diff
@@ -122,6 +132,16 @@
...
       </xmpp:SupportedXep>
     </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0166.html" />
+      </xmpp:SupportedXep>
+    </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0153.html" />
```
[the remark this answers is not in the document]
Comment (Zash) [reply dino.doap.in:142]: Ack, fixed locally.

File: dino.doap.in (hunk 1 above)
Comment (Zash) [reply dino.doap.in:289]: Shall I put them in as "planned" or drop that commit?

Comment (fiaxh) [reply dino.doap.in:296]: I'd prefer to just leave them out for now.

File: dino.doap.in [hunk 3]
```diff
@@ -122,12 +132,27 @@
...
       </xmpp:SupportedXep>
     </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0166.html" />
+      </xmpp:SupportedXep>
+    </implements>
+    <implements>
+      <xmpp:SupportedXep>
+        <xmpp:xep rdf:resource="https://xmpp.org/extensions/xep-0166.html" />
```
Comment (fiaxh) [inline dino.doap.in:153]: XEP-0166 is listed twice.

Comment (Zash) [reply dino.doap.in]: Fixed

File: dino.doap.in (hunk 1 above)
Comment (Zash) [reply dino.doap.in:291]: Ok, dropping that commit.

Comment (Zash) [reply dino.doap.in:291]: ... apparently I dropped the wrong commit. git :cry:

Comment (Zash) [reply dino.doap.in]: Now then?

**Comment by fiaxh** [comment]
Thanks!

**Pull request merged**